src/custom/theme/styled.d.ts

```diff
@@ -1,5 +1,5 @@
-import { Colors as ColorsUniswap } from '@src/theme/styled'
-export { Color, Grids } from '@src/theme/styled'
+import { Colors as ColorsUniswap } from '@src/theme/styled.d'
+export { Color, Grids } from '@src/theme/styled.d'
```

Comment:
With addition of `package.json::jest` extensions stopped resolving to `d.ts` automatically, when running tests.
And cra/craco only allows certain jest config params to be changed, so no `moduleFileExtensions` additions.
Had to specify here.